problem_builder/tests/integration/base_test.py

```diff
@@ -57,7 +60,18 @@ def popup_check(self, mentoring, item_feedbacks, prefix='', do_submit=True):
             self.assertFalse(item_feedback_popup.is_displayed())
 
 
-class MentoringAssessmentBaseTest(MentoringBaseTest):
+class MentoringBaseTest(SeleniumBaseTest, PopupCheckMixin):
+    module_name = __name__
+    default_css_selector = 'div.mentoring'
```

Comment:
@e-kolpakov True, but I don't like referencing a base class before it's defined.

Also, `MentoringAssessmentBaseTest` should not be a child of `MentoringBaseTest`, because `MentoringBaseTest` inherits the `setUp` method from `SeleniumBaseTest` which we don't want (It parses and loads *all* of the integration test .xml files for every test, and starts each test by loading the workbench home page - we don't really want either of those to be happening since they pointlessly slow down the tests).